PHPCompatibility/Sniffs/Constants/RemovedConstantsSniff.php

```diff
@@ -1933,7 +1933,12 @@ class RemovedConstantsSniff extends Sniff
             'extension' => 'tokenizer',
         ],
         'T_BAD_CHARACTER' => [
-            '7.0'       => true,
+            '5.6' => false,
+            '7.0' => true,
+            '7.1' => true,
+            '7.2' => true,
+            '7.3'       => true,
+            '7.4'       => false,
```

Comment:
This change breaks the functionality of the `ComplexVersionDeprecatedRemovedFeatureTrait::getVersionInfo()` method as this array is expected to only have a `false` value _before_ a `true` value.

Reply:
Nothing in the test suite complained about this...

Reply:
I wouldn't, but only because you did the special casing (and hard `return`) _above_ the logic in the sniff which used the method from the trait.

It is the reason the original special casing needed so much extra logic as the results from the trait method were no longer usable.